pkg/common/security/acl.go

```diff
@@ -123,10 +123,6 @@ func NewACL(aclStr string) (ACL, error) {
 	}
 	// trim and check for wildcard
 	acl.setAllAllowed(aclStr)
-	// an empty ACL is a deny
```

Comment:
The above code `fields := strings.Split(aclStr, common.Space)` will must lead to `fields` length more than 0
So it will be unreachable for the condition expression.